Bender.yml

```diff
@@ -22,10 +22,12 @@ sources:
   # package. Files in level 1 only depend on files in level 0, files in level 2 on files in
   # levels 1 and 0, etc. Files within a level are ordered alphabetically.
   # Level 0
+  - src/axi_burst_splitter_table.sv
   - src/snitch_icache_pkg.sv
   - src/riscv_instr_branch.sv
   - src/multi_accept_rr_arb.sv
   # Level 1
+  - src/l0_to_bypass.sv
```

Comment:
I believe this should be in level 0, or on what level 0 file does it depend on?

Reply:
AFAIK it depends on `snitch_icache_pkg`

Reply:
Good point yes.